package.json

```diff
@@ -24,6 +25,7 @@
   "author": "Gaidarenko Oleg <markelog@gmail.com>",
   "license": "MIT",
   "devDependencies": {
+    "assert-diff": "^1.2.0",
```

Comment:
Using assert-diff as otherwise the results of `assert.deepEqual` failing is next to useless.